packages/miniflare/turbo.json

```diff
@@ -3,12 +3,10 @@
 	"extends": ["//"],
 	"tasks": {
 		"build": {
-			"inputs": ["src/**", "scripts/**", "types/**", "*.mjs", "*.js", "*.json"],
```

Comment:
The only slightly annoying thing about this is that miniflare is a little slow to build and if you are just making changes to tests the build will have to re-run each time. But I can live with that.

Reply:
I've refactored this slightly so that changes to only test files shouldn't cause the build to re-run